contributing/code/bc.rst

```diff
@@ -234,134 +177,96 @@ Changing Classes
+Move to parent class                                Yes
 **Protected Properties**
-Add protected property                              Yes             Yes
-Remove protected property                           Yes [2]_        No
-Reduce visibility                                   Yes [2]_        No
-Move to parent class                                Yes             Yes
+Add protected property                              Yes
+Remove protected property                           No
+Reduce visibility                                   No
+Move to parent class                                Yes
 **Private Properties**
-Add private property                                Yes             Yes
-Remove private property                             Yes             Yes
+Add private property                                Yes
+Remove private property                             Yes
 **Constructors**
-Add constructor without mandatory arguments         Yes [2]_        Yes [2]_
-Remove constructor                                  Yes [2]_        No
-Reduce visibility of a public constructor           No              No
-Reduce visibility of a protected constructor        Yes [2]_        No
-Move to parent class                                Yes             Yes
+Add constructor without mandatory arguments         Yes [1]_
+Remove constructor                                  No
+Reduce visibility of a public constructor           No
+Reduce visibility of a protected constructor        No
+Move to parent class                                Yes
 **Public Methods**
-Add public method                                   Yes             Yes
-Remove public method                                No              No
-Change name                                         No              No
-Reduce visibility                                   No              No
-Move to parent class                                Yes             Yes
-Add argument without a default value                No              No
-Add argument with a default value                   Yes [2]_        No
-Remove argument                                     Yes [4]_        Yes [4]_
-Add default value to an argument                    Yes [2]_        No
-Remove default value of an argument                 No              No
-Add type hint to an argument                        Yes [8]_        No
-Remove type hint of an argument                     Yes [2]_        No
-Change argument type                                Yes [2]_ [5]_   No
-Change return type                                  Yes [2]_ [6]_   No
+Add public method                                   Yes
+Remove public method                                No
+Change name                                         No
+Reduce visibility                                   No
+Move to parent class                                Yes
+Add argument without a default value                No
+Add argument with a default value                   No
```

Comment:
This was the only option we had as PHP will complain anyway (see https://3v4l.org/6T2DO).